src/AzureIoTHub.Portal.Domain/Entities/AWS/ThingType.cs

```diff
@@ -11,6 +11,7 @@ public class ThingType : EntityBase
         [Required]
         public string Name { get; set; } = default!;
         public string? Description { get; set; } = default!;
+        public bool deprecated { get; set; }
```

Comment:
Should be PascalCase
```suggestion
        public bool Deprecated { get; set; }
```